stripe-core/src/main/java/com/stripe/android/core/utils/Encode.kt

```diff
@@ -14,29 +15,34 @@ private val json = Json {
     encodeDefaults = true
 }
 
-internal fun b64Encode(s: String): String =
+fun b64Encode(s: String): String =
```

Comment:
Should we add `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP_PREFIX)` to these?

Reply:
ahh probably